config.in

```diff
@@ -24,6 +24,11 @@ set $menu dmenu_path | dmenu | xargs swaymsg exec --
 # window corner radius in px
 corner_radius 10
 
+# inactive window fade amount. 1.0 = no dimming, 0.0 = fully dimmed
+dim_inactive 1.0
```

Comment:
I think we should think of a better name here, since `dim_inactive 1.0` would seem to suggest that inactive windows are dimmed. Perhaps `inactive_brightness`? Open to other's thoughts on wording here!

Reply:
Hmm... I have a few on top of my head, `inactive_shade`, `inactive_blend`, and `inactive_blend`? Maybe take this to the Discord server?